recipes/auto_gptq/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "auto_gptq" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://github.com/PanQiWei/AutoGPTQ/archive/v{{ version }}.tar.gz
+  sha256: 580de955332cb06fca287ee137ea6aa15f56ce402485f01d6161b0a976b83cc5  # 0.6.0
+
+build:
+  # noarch: python
+  script_env:
+    - DISABLE_QIGEN=1 # coming soon
+    - TORCH_CUDA_ARCH_LIST=3.5;5.0;6.0;6.1;7.0;7.5;8.0;8.6+PTX      # [cuda_compiler_version == "11.2"]
+    - TORCH_CUDA_ARCH_LIST=3.5;5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9+PTX  # [cuda_compiler_version == "11.8"]
+    - TORCH_CUDA_ARCH_LIST=5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX  # [(cuda_compiler_version or "").startswith("12")]
+    - BUILD_CUDA_EXT=0
+  script:
+    |
+      cat setup.py | sed 's/os.environ\["CC"\] = "g++"//' | sed 's/os.environ\["CXX"\] = "g++"//' > tmpsetup.py
+      mv tmpsetup.py setup.py
+      {{ PYTHON }} -m pip install . -vv --no-cache-dir auto-gptq=={{ version }}
+  number: 0
+  skip: True  # [cuda_compiler_version == "None"]
+  skip: True  # [osx or win]
+  missing_dso_whitelist:
+    - '*/libtorch_python.so'
+    
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+
+  host:
+    - python
```

This will help align CUDA package versions between `build` & `host`, which should cleanup the CI issues

```suggestion
  host:
    - cuda-version {{ cuda_compiler_version }}
    - python
```